docs/topics/forms/modelforms.txt

```diff
@@ -101,6 +101,13 @@ Model field                         Form field
 
 :class:`GenericIPAddressField`      :class:`~django.forms.GenericIPAddressField`
 
+:class:`JSONField`                  :class:`~django.forms.JSONField` with
+                                    :attr:`~django.forms.JSONField.encoder` and
+                                    :attr:`~django.forms.JSONField.decoder` set
+                                    to the model field's
+                                    :attr:`~JSONField.encoder` and
+                                    :attr:`~JSONField.decoder`.
```

Comment:
I would chop note about encoders/decoders, it's already mentioned in the `django.forms.JSONField` docs.
```suggestion
:class:`JSONField`                  :class:`~django.forms.JSONField`
```

Reply:
That's how I initially wrote this PR :laughing:

Okay, done.